exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,24 @@
+
 var mentor1 = "Daniel";
 var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function uppercaseName(name){
+    var updatedName = name.toUpperCase();
+    return updatedName;
+}
+
+function shouti(mentor){
+    var mentorName = uppercaseName(mentor);
+    var message = "Hello " + mentorName;
+    console.log(message);
+}
```

Comment:
You have clearly understood how to call one function from another, and again, you have used goo variable names.